src/lib/people/team/team.ts

```diff
@@ -0,0 +1,47 @@
+import * as v from 'valibot';
+
+export const committees = [
+    'Executive',
+    'Innovation',
+    'Service',
+    'External Relations',
+    'Membership & Internals',
+    'Branding & Creatives',
+    'Engineering',
+];
+
+const MemberSchema = v.object({
+    name: v.string(),
+    title: v.string(),
+    committee: v.pipe(
+        v.string(),
+        v.check(i => committees.includes(i), 'invalid committee'),
+    ),
+    socials: v.optional(v.record(v.string(), v.pipe(v.string(), v.url()))),
+});
+
+export interface Member extends v.InferOutput<typeof MemberSchema> {
+    img: string;
+}
+
+const baseRe = /(?<base>\w+).json$/;
+
+export async function getTeam(): Promise<Member[]> {
+    const imports = import.meta.glob<Member>('./members/*.json');
+
+    const promises = Object.entries(imports).map(async ([path, asset]) => {
+        // Derive the image name from the original path
+        // It goes without saying that this may not be the right approach for this
+        const base = path.match(baseRe)?.groups?.base;
+        const img: string = (await import(`$lib/people/team/images/${base}.webp`)).default;
```

Comment:
We can probably do something cute like this just to avoid the weird-looking parentheses.
```suggestion
        const { default as img } = await import(`$lib/people/team/images/${base}.webp`);
```

Reply:
For now, I won't be changing this part (besides adding `?url` to the import) or applying your suggestions here due to the aforementioned [type inferencing issues mentioned in a previous reply](https://github.com/up-csi/up-csi.github.io/pull/92#discussion_r1799220272).